servus/uint128_t.h

```diff
@@ -355,7 +391,31 @@ struct hash< servus::uint128_t >
         hash< uint64_t > forward;
         return forward( in.high( )) ^ forward( in.low( ));
     }
+    result_type operator()( const servus::uint128_t& in )
+    {
+        hash< uint64_t > forward;
+        return forward( in.high( )) ^ forward( in.low( ));
+    }
```

Comment:
It's not needed. I was having a compilation error related to the namespace and I misunderstood the "clearly stated" error message. I forgot to remove the overload once I found the error.